content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -202,10 +197,39 @@ The header and payload of the `TOKEN` may be examined with the following command
 
 ```shell
 # Header
-echo -n $TOKEN | cut -d '.' -f 1 | base64 -D
+echo -n $TOKEN | cut -d '.' -f 1 | base64 -d | jq
+```
+
+Generates output similar to:
+
+```json
+{
+  "alg": "RS256",
+  "typ": "at+jwt",
+  "kid": "devidp"
+}
+```
 
+```shell
 # Payload
-echo -n $TOKEN | cut -d '.' -f 2 | base64 -D
+echo -n $TOKEN | cut -d '.' -f 2 | base64 -d | jq
```

Comment:
Split the header and payload out to separate copy/paste commands. Although, with the jq output, it might be ok to merge back to one block copy/paste